### DISCUSSION TITLE
Could not create Bug on Test Pass

### DISCUSSION THREAD

**Discussion by ShouptikDeyIllumina** [axios/axios#5856] [Q&A]
### Describe the issue

Hi,
   I am trying to create a playwright custom reporter to Create Bug Whenever there is a failure in Test cases. I have wrote a method called CreateBug(). This method is creating but using post call. But i am facing a wired problem. Besically after a test has been executed it run onTestEnd() method. So i am seeing 3 types of behaviors: 
1> when both (Test A and Test B)passed -> No bug created 
2> When Test A  passed but Test B failed -> its creating Bug for the passed test(Test A)
3> When Both Test Failed -> No Bug Created.

In pic A both test passed but just for experiment i am not using any if here insted i am calling CreareBug() method. and method is being called as well. i have used a debug.log() in Createbug method just before Post method. as we can see in the terminal its printing the debug.log but after that post method didn't worked. If any error occurred try block should have caught it and print the error on catch block. that also is not coming. i believe for some reason is post method is being called not sure why 

Please can anyone help me for this issue?

**Pic A**
![image](https://github.com/axios/axios/assets/135975495/bd0b668b-613f-40a9-b3e7-85fdf37a155d)

**Comment by ShouptikDeyIllumina** [comment]
Note That Some time (For Case 2) A bug is getting created

**Comment by jasonsaayman** [comment]
This is far out of scope i think. Maybe better for stackoverflow?
